### PR TITLE
Dynamic landing page

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: dash/orgs/middleware.py [hunk 1]
```diff
@@ -74,14 +77,13 @@ def process_view(self, request, view_func, view_args, view_kwargs):
...
-            if not url_name in whitelist:
-                all_orgs = Org.objects.filter(is_active=True).order_by('name')
-
-                # populate a 'host' attribute on each org so we can link off to them
-                for org in all_orgs:
-                    org.host = settings.SITE_HOST_PATTERN % org.subdomain
+            # make sure the chooser view is whitelisted
+            chooser_view = getattr(settings, 'SITE_CHOOSER_VIEW_NAME', None)
+            if chooser_view:
+                whitelist += (chooser_view,)
```
Comment (nicpottier) [inline dash/orgs/middleware.py]: Shouldn't this default to orgs.org_chooser like below? (maybe pull this out as a variable for both places)

File: dash/orgs/migrations/0010_auto_20150414_1432.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('orgs', '0009_auto_20150331_1452'),
+    ]
+
+    operations = [
+        migrations.AddField(
```
Comment (nicpottier) [inline dash/orgs/migrations/0010_auto_20150414_1432.py]: Again, this probably doesn't belong in dash, as the concept of flags is something specific to U-Report.

Maybe this is another type of image like background and banner?

File: dash/orgs/middleware.py (hunk 1 above)
Comment (norkans7) [inline dash/orgs/middleware.py]: Right I adjust this

File: dash/orgs/migrations/0010_auto_20150414_1432.py (hunk 2 above)
Comment (norkans7) [inline dash/orgs/migrations/0010_auto_20150414_1432.py]: Ok, though I realize now that the Orgbackground are part of dash already

**Comment by nicpottier** [comment]
Looks good!

**Pull request merged**